### DISCUSSION TITLE
How can I get sea-orm-migration to print the migration SQL?

### DISCUSSION THREAD

**Discussion by thesecretmaster** [SeaQL/sea-orm#1664] [Q&A] [answered]
I'm writing migration using sea-orm-migration. The migration doesn't use any raw sql anywhere, but it still fails with `Execution Error: error returned from database: syntax error at or near "("`. I have no way to debug this without being able to see the generated sql, and even passing the `-v` flag to `sea migration up -v` doesn't print it out. I'm on sea-orm-migration 0.11. How can I resolve this?

The `-v` flag does seem to work in 0.10.7 (I tried downgrading) but breaks in 0.11.0.

EDIT: I've resolved the issue by seeing the SQL printed in 0.10.7, but leaving this open because I think it may be a bug/regression. I also don't want to lock to 0.10 forever. Also a note -- the docs indicate that migrations are all-or-nothing, but when these migrations fail with a SQL syntax error they do not roll back the previous changes.

**Comment by tyt2y3** [comment]
Did you try setting an env var `export RUST_LOG=info` ?

**Reply by thesecretmaster** [reply]
Yep, still doesn't print anything (in 0.11)

**Comment by billy1624** [comment, marked as answer]
Hey @thesecretmaster, you will need to do two things.

1. Add SeaORM dependency and enable `debug-print`, `sea-orm = { version = "0.11", features = ["debug-print"] }`
2. Enable verbose mode `sea-orm-cli migrate -v`

**Reply by thesecretmaster** [reply]
```
error: failed to select a version for the requirement `sea-orm = "^0.12.0"`
```

Looking at `crates.io` it doesn't look like there is a 0.12 of `sea-orm` out yet?

**Reply by billy1624** [reply]
Sorry it should be `0.11`
